tools/tune/crossover/crossover_plot_freq.m

```diff
@@ -18,6 +18,8 @@
 % Then to plot the transferfn for y1 we would create a filter such as:
 %  x(n) ---> LR4 LO-PASS --> LR4 LO-PASS --> y1(n)
 
+figure;
+
```

Comment:
At the line number 71 and 72, `w` , please consider renaming as variable is over written and unused. at the same time , please consider removing semicolon at line `68` end .